.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 4.2.0
+current_version = 4.3.0
```

Comment:
It would have been better to first review, then merge and then bump. Since there is only a single pull request open right now, then this is not an issue, thus no need to revert. But you need to push the git tag so that the package is pushed to pypi.

Reply:
i completely agree